### ISSUE TITLE
Help with Docker image applying

### ISSUE THREAD

**Issue by Aphexus** [Javernaut/ffmpeg-android-maker#96]
I've never really used docker but following the info when I run I get

/bin/sh: 1: cd: can't cd to /mnt/ffmpeg-android-maker

Viewing the files there is no such directory

I tried to build in wsl but after installing android studio I and building I got some error about clang not being able to build the images.

Best I can tell is the doctor image doesn't contain ffmpeg-android-maker or it's in a different folder. (tried to find it in the docker files list but nothing)

**Comment by Javernaut** [comment]
Hello @Aphexus 

The Docker image indeed doesn't contain the ffmpeg-android-maker directory on its own. The sole point of the Docker image I created is to have preinstalled all the necessary software for building FFmpeg inside of it. But the 'ffmpeg-andorid-maker' directory is supplied to the image from the host machine. This gives you an opportunity to modify the scripts in the project and still use the same Docker image for building it.

Read [this section](https://github.com/Javernaut/ffmpeg-android-maker/wiki/Docker-support#run-docker-run) about what parameters that you can use for executing the image. The whole document contains workable examples.

If things still don't work, tell me what command do you use for launching the Docker image?

**Comment by Aphexus** [comment]
Ok, I thought that might be the case but when I ran the commands and followed the help it would exit. Initially there was no info in the log. Basically I followed the help and such(downloaded maker, corrected path info, etc) but maybe I missed something. I then updated docker because I haven't messed with it for a while then it started showing the error I gave I assumed it was missing. I'll try again.

**Comment by Aphexus** [comment]
Ok, I was eventually able to get things to work. Not sure what the problem was. I think I might have got the path wrong or it was some issue with docker. I did have some issues compiling with all the extensions added though so I had to remove them.

The command that I started with

docker run --rm -v S:\Temp\c\javernaut\ffmpeg-android-maker:/mnt/ffmpeg-android-maker -e FAM_ARGS="-aom -bluray -freetype -fribidi -opus -speex  -webp -x264 -vpx -lame -android=24" javernaut/ffmpeg-android-maker

and I had to to remove several but I don't remember. I think it was bluray, freetype, vpx, fribidi, and lame. I didn't investigate much into the reasons it didn't work but something about clang IIRC. If I build it again, since now I know it works, I'll pay attention.
